x-pack/plugins/apm/public/components/app/error_group_details/top_erroneous_transactions/index.tsx

```diff
@@ -0,0 +1,264 @@
+        return callApmApi(
+          'GET /internal/apm/services/{serviceName}/errors/{groupId}/top_erroneous_transactions',
+          {
+            params: {
+              path: {
+                serviceName,
+                groupId,
+              },
+              query: {
+                environment,
+                kuery,
+                start,
+                end,
+                numBuckets: 20,
+                offset:
+                  comparisonEnabled && isTimeComparison(offset)
+                    ? offset
+                    : undefined,
+              },
+            },
+          }
+        );
+      }
+    },
+    [
+      environment,
+      kuery,
+      serviceName,
+      start,
+      end,
+      groupId,
+      comparisonEnabled,
+      offset,
+    ]
+  );
+
+  const loading =
+    status === FETCH_STATUS.LOADING || status === FETCH_STATUS.NOT_INITIATED;
+
+  const addKueryBarFilter = ({ key, value }: { key: string; value: any }) => {
+    pushNewItemToKueryBar({ kuery, history, key, value });
+  };
+
+  const columns: Array<
+    EuiBasicTableColumn<
+      ValuesType<ErroneousTransactions['topErroneousTransactions']>
+    >
+  > = [
+    {
+      field: 'transactionName',
+      width: '40%',
+      name: i18n.translate(
+        'xpack.apm.errorGroupTopTransactions.column.transactionName',
+        {
+          defaultMessage: 'Transaction name',
+        }
+      ),
+      render: (_, { transactionName, transactionType }) => {
+        return (
+          <>
+            <EuiButtonEmpty
+              onClick={() => {
+                addKueryBarFilter({
+                  key: TRANSACTION_NAME,
+                  value: transactionName,
+                });
+              }}
+              data-test-subj={`filter_by_transaction_name`}
+            >
+              <EuiToolTip
+                position="top"
+                content={i18n.translate(
+                  'xpack.apm.errorGroupTopTransactions.actionFilterLabel',
+                  { defaultMessage: 'Filter by value' }
+                )}
+              >
+                <EuiIcon type="filter" color="text" size="m" />
+              </EuiToolTip>
+            </EuiButtonEmpty>
+            <TruncateWithTooltip
+              text={transactionName}
+              content={
+                <TransactionDetailLink
+                  serviceName={serviceName}
+                  transactionName={transactionName}
+                  transactionType={transactionType ?? ''}
+                  comparisonEnabled={comparisonEnabled}
+                  offset={offset}
+                >
+                  {transactionName}
+                </TransactionDetailLink>
+              }
+            />
+          </>
+        );
+      },
+    },
+    {
+      field: 'occurrences',
+      name: i18n.translate(
+        'xpack.apm.errorGroupTopTransactions.column.occurrences',
+        {
+          defaultMessage: 'Number of transactions affected',
```

Comment:
Is this the number of transactions affected or the number of occurrences? (Multiple errors can occur in the scope of a single transaction).

Reply:
It's the number of error occurrences for a particular transaction name.

Reply:
I think the label is incorrect then?

Reply:
That's right good spot. It should be something like "Occurrences" as in the other error tables.